python/dgl/nn/pytorch/conv/appnpconv.py

```diff
@@ -80,6 +80,9 @@ def forward(self, graph, feat):
         feat : torch.Tensor
             The input feature of shape :math:`(N, *)`. :math:`N` is the
             number of nodes, and :math:`*` could be of any shape.
+        edge_weight: Optional[torch.Tensor]
+            edge_weight used in the message passing process. Default to th.ones
+            if not specified
```

Comment:
How will this affect the equation above? Replace the binary adjacency matrix $A$ with a weighted one?